src/xenia/gpu/graphics_system.cc

```diff
@@ -187,8 +201,6 @@ uint32_t GraphicsSystem::ReadRegister(uint32_t addr) {
       return 0x08100748;
     case 0x0F01:  // RB_BC_CONTROL
       return 0x0000200E;
-    case 0x194C:  // R500_D1MODE_V_COUNTER
-      return 0x000002D0;
```

Comment:
If we need this to provide a value that changes at some rate, we need to read the system clock at register reads and to remap it to the expected frequency, not to take 100% of one host CPU core.